report_a_breach/templatetags/gtm_enabled.py

```diff
@@ -0,0 +1,10 @@
+from django import template
+from django.conf import settings
+
+register = template.Library()
+
+
+# settings value
+@register.simple_tag
+def gtm_enabled():
+    return settings.GTM_ENABLED
```

Comment:
I wonder if we can also check for the presence of the container ID here as one is useless without the other, e.g.

```suggestion
    return settings.GTM_ENABLED and settings.GTM_ID
```